scripts/cli.py

```diff
@@ -361,8 +390,8 @@ def microk8s():
     default=['dns', 'storage', 'dashboard', 'ingress', 'metallb:10.64.140.43-10.64.140.49'],
     multiple=True,
 )
-@click.option('--model-defaults', default=[], multiple=True)
-def setup(controller, services, model_defaults):
+@click.option('--test-mode/--no-test-mode', default=False)
+def setup(controller, services, test_mode):
```

Comment:
Hrm.  So we lose the ability to specify other arbitrary defaults, but it becomes more consistent and obvious.  And this is already a convenience helper anyway, so you could always do it yourself if you want more control.  Seems reasonable.